sdk/appcenter-crashes/src/androidTest/java/com/microsoft/appcenter/crashes/WrapperSdkExceptionManagerAndroidTest.java

```diff
@@ -109,7 +109,10 @@ public void saveWrapperException() {
             startFresh();
 
             /* Save crash. */
-            error.id = WrapperSdkExceptionManager.saveWrapperException(Thread.currentThread(), null, new Exception(), error.data);
+            Exception exception = new Exception();
+            exception.setType("type");
```

Comment:
What is the reason for those placeholder values for type and message?

Reply:
reverted